### PR TITLE
[Tizen/Web] Refactoring ImageClassificationOffloading example

### PR DESCRIPTION
**Pull request by niley7464**
- Create utility module
- Functionalization of duplicated part
- Remove unused variables
- Change var variables to use let type
- Declare variables closer to where they are used

### CODE REVIEW

File: Tizen.web/ImageClassificationOffloading/js/main.js [hunk 1]
```diff
@@ -66,109 +77,37 @@ function runLocal() {
...
-        const label = document.getElementById('label_offloading');
-        const tensorsRetData = data.getTensorRawData(0);
-        const maxIdx = GetMaxIdx(tensorsRetData.data);
-        label.innerText = labels[maxIdx];
-
-        const time = document.getElementById('time_offloading');
-        time.innerText = 'offloading : ' + (endTime - startTime) + ' ms'
-    });
-}
+async function runOffloading() {
```
Comment (yeonykim2) [inline Tizen.web/ImageClassificationOffloading/js/main.js:92]: The `ip` is not set in `runOffloading()` func. So, it can be removed `async`.

**Review by yeonykim2 — CHANGES_REQUESTED** [review]

File: Tizen.web/ImageClassificationOffloading/js/main.js (hunk 1 above)
Comment (niley7464) [reply Tizen.web/ImageClassificationOffloading/js/main.js]: I missed that part during the refactoring process.
Thank you :)

**Review by myungjoo — APPROVED** [review]

**Review by wooksong — APPROVED** [review]

File: Tizen.web/ImageClassificationOffloading/js/utils.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    var max = array[0];
+    var maxIdx = 0;
+
+    for (var i = 0; i < array.length; ++i) {
+        if (array[i] > max) {
+            maxIdx = i;
+            max = array[i];
+        }
+    }
+    return maxIdx;
```
Comment (wooksong) [inline Tizen.web/ImageClassificationOffloading/js/utils.js:58]:
```suggestion
    let max = array[0];
    let maxIdx = 0;

    for (let i = 0; i < array.length; ++i) {
        if (array[i] > max) {
            maxIdx = i;
            max = array[i];
        }
    }
    return maxIdx;
```

Comment (wooksong) [reply Tizen.web/ImageClassificationOffloading/js/utils.js:58]:
```suggestion
    let maxIdx = array.indexOf(Math.max(...array));

    return maxIdx;
```

File: Tizen.web/ImageClassificationOffloading/js/utils.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+/**
+ * Get the jpeg image path
+ * @returns image path
+ */
+export function GetImgPath() {
+    const MAX_IMG_CNT = 2;
+    var imgsrc = GetImgPath.count++ % MAX_IMG_CNT;
```
Comment (wooksong) [inline Tizen.web/ImageClassificationOffloading/js/utils.js:67]:
```suggestion
    let imgsrc = GetImgPath.count++ % MAX_IMG_CNT;
```

File: Tizen.web/ImageClassificationOffloading/js/utils.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+}
+GetImgPath.count = 0;
+
+/**
+ * Load the label from the text file and return the string array
+ * @returns string array
+ */
+export function loadLabelInfo() {
+    var fHandle = tizen.filesystem.openFile('wgt-package/res/labels.txt', 'r');
+    var labelList = fHandle.readString();
```
Comment (wooksong) [inline Tizen.web/ImageClassificationOffloading/js/utils.js:79]:
```suggestion
    const fHandle = tizen.filesystem.openFile('wgt-package/res/labels.txt', 'r');
    const labelList = fHandle.readString();
```

File: Tizen.web/ImageClassificationOffloading/js/utils.js (hunk 2 above)
Comment (niley7464) [reply Tizen.web/ImageClassificationOffloading/js/utils.js]: I divided the modules and forgot to change util.js 😅
Thank you :)

File: Tizen.web/ImageClassificationOffloading/js/utils.js (hunk 3 above)
Comment (niley7464) [reply Tizen.web/ImageClassificationOffloading/js/utils.js]: Updated! Thank you

File: Tizen.web/ImageClassificationOffloading/js/utils.js (hunk 4 above)
Comment (niley7464) [reply Tizen.web/ImageClassificationOffloading/js/utils.js]: Updated! Thank you

File: Tizen.web/ImageClassificationOffloading/js/utils.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
+/* SPDX-License-Identifier: Apache-2.0-only */
```
Comment (myungjoo) [inline Tizen.web/ImageClassificationOffloading/js/utils.js:1]: ```s/SPDX-License-Identifier: Apache-2.0-only/SPDX-License-Identifier: Apache-2.0/```

**Review by myungjoo — COMMENTED** [review]
Update SPDX identifiers.

There is no "Apache-2.0-only". Use "Apache-2.0".
LGPL-2.0-only has some context...

File: Tizen.web/ImageClassificationOffloading/js/utils.js (hunk 5 above)
Comment (niley7464) [reply Tizen.web/ImageClassificationOffloading/js/utils.js]: Updated! Thank you :)

**Pull request merged**